config/custom-formats.ts

```diff
@@ -0,0 +1,5 @@
+// Note: This is the list of formats
+// The rules that formats use are stored in data/rulesets.js
```

Comment:
```suggestion
// The rules that formats use are stored in data/rulesets.ts
```

Reply:
I copied that from formats. I will update that as well